_posts/2020-12-28-subdomains.md

```diff
@@ -0,0 +1,39 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: European Galaxy Flavours
+tags: []
+supporters:
+---
+
+[UseGalaxy.eu](https://usegalaxy.eu/) hosts several domain-specific versions of the usegalaxy.eu server. These domains are __featured interfaces__ that show a subset of the analysis tools, datasets and workflows relevant for different research areas:
+
+
+ RESEARCH AREA |  DOMAIN 
+-------------- | --------: 
+African Research | [africa.usegalaxy.eu](https://africa.usegalaxy.eu)
+Galaxy for Genome Annotation | [annotation.usegalaxy.eu](https://annotation.usegalaxy.eu)
+Computational Chemistry | [cheminformatics.usegalaxy.eu](https://cheminformatics.usegalaxy.eu)
+Climate Science | [climate.usegalaxy.eu](https://climate.usegalaxy.eu)
+CLIP-Explorer | [clipseq.usegalaxy.eu](https://clipseq.usegalaxy.eu)
+COVID-19 | [covid19.usegalaxy.eu](https://covid19.usegalaxy.eu)
+Ecology | [ecology.usegalaxy.eu](https://ecology.usegalaxy.eu)
+Erasmus MC | [erasmusmc.usegalaxy.eu](https://erasmusmc.usegalaxy.eu)
+Clustering of RNA | [graphclust.usegalaxy.eu](https://graphclust.usegalaxy.eu)
+HiCExplorer | [hicexplorer.usegalaxy.eu](https://hicexplorer.usegalaxy.eu)
+Human Cell Atlas | [humancellatlas.usegalaxy.eu](https://humancellatlas.usegalaxy.eu)
+Imaging | [imaging.usegalaxy.eu](https://imaging.usegalaxy.eu)
+Metabolomics | [metabolomics.usegalaxy.eu](https://metabolomics.usegalaxy.eu)
+Metagenomics | [metagenomics.usegalaxy.eu](https://metagenomics.usegalaxy.eu)
+Machine Learning | [ml.usegalaxy.eu](https://ml.usegalaxy.eu)
+NanoGalaxy | [nanopore.usegalaxy.eu](https://nanopore.usegalaxy.eu)
+Plant Biology | [plants.usegalaxy.eu](https://plants.usegalaxy.eu)
+Proteomics | [proteomics.usegalaxy.eu](https://proteomics.usegalaxy.eu)
+RNA | [rna.usegalaxy.eu](https://rna.usegalaxy.eu)
+Single Cell Omics | [singlecell.usegalaxy.eu](https://singlecell.usegalaxy.eu)
+Street Science Community | [streetscience.usegalaxy.eu](https://streetscience.usegalaxy.eu)
+Virology | [virology.usegalaxy.eu](https://virology.usegalaxy.eu)
+{: .table.table-striped }
+
+
+
+All the tools available in the different domains are also available in [UseGalaxy.eu](usegalaxy.eu) and the __quota__ and data associated with the user account are shared through all the domains.
```

Comment:
I would move this up, this is the important message that we want to convey, left? 

Maybe also a sentence that we are happy to add more subdomains if someone wants to maintain their own subdomain.